cli/cmd/grafanaChangePassword.go

```diff
@@ -45,10 +45,27 @@ func grafanaChangePassword(cmd *cobra.Command, args []string) error {
 
 	err = k8s.KubeExecCmd(namespace, grafanaPod, "grafana", "grafana-cli admin reset-admin-password "+password, nil, false)
 	if err != nil {
-		secret.Data["admin-password"] = oldpassword
-		_ = k8s.KubeUpdateSecret(namespace, secret)
-		return fmt.Errorf("could not change Grafana password: %w", err)
+		err1 := updateToOldPassword(oldpassword)
```

Comment:
Noticed an issue in this use-case, If the new password is short exmaple `hii` pwd updation fails from grafana stating pwd is too short. But here our fallback mechanism is trying to update back to the old pwd by updating the secret object we hold from the previous get operation, This is retrieved before updating the new password. 

This updation fails as the secret object we are trying to assign is too old and metadata of the secret doesn't let us use the object to update pwd. So we need to retrieve again the pwd for updataion.